frontend/seo/matrix.ts

```diff
@@ -0,0 +1,43 @@
+import { useDBInstanceStore } from "../src/stores/dbInstance";
+import { CloudProvider } from "../src/types";
+
+export const cloudProviderList: string[] = ["AWS", "GCP"];
```

Comment:
We may need to define a structure with name and id, where id is used as the URL.